modules/auxiliary/admin/mssql/mssql_enum_domain_accounts.rb

```diff
@@ -30,13 +30,20 @@ def initialize(info = {})
     register_options(
```

Comment:
I'm going to guess that this and the `set_sane_defaults` will get offloaded to a library somewhere?

Reply:
Yep, idea is to have this data passed to the client class when it's created